docs/source/multidc/multidc.md

```diff
@@ -586,12 +586,12 @@ auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqns
 
 Save the output, replace the token with your own, and patch the secret in the second datacenter with the below command:
 ```shell
-kubectl --context="${CONTEXT_DC2}"-n=scylla patch secret/scylla-cluster-auth-token --type='json' -p='[{"op": "replace", "path": "/data/auth-token.yaml", "value": "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf"}]'
+kubectl --context="${CONTEXT_DC2}" -n=scylla patch secret/scylla-cluster-auth-token --type=json --patch="[{\"op\": \"replace\", \"path\": \"/data/auth-token.yaml\", \"value\": \"$( echo "auth_token: 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf" | base64 --wrap=0 )\" }]"
```

Comment:
otherwise this avoids backslashes
```bash
kubectl --context="${CONTEXT_DC2}"-n=scylla patch secret/scylla-cluster-auth-token --type='json' -p='[{"op": "replace", "path": "/data/auth-token.yaml", "value": "auth_token: '"$( echo 84qtsfvm98qzmps8s65zr2vtpb8rg4sdzcbg4pbmg2pfhxwpg952654gj86tzdljfqnsghndljm58mmhpmwfgpsvjx2kkmnns8bnblmgkbl9n8l9f64rs6tcvttm7kmf | base64 --wrap=0 )"'"}]'
```

Reply:
I've used the backslashes for consistency with the other command, but I don't mind either.

Reply:
done